src/components/ComposerQuill/utils.js

```diff
@@ -169,9 +193,9 @@ export function buildMentionAvatar(item) {
     // otherwise we build it ourself
     let initials;
 
-    if (id === 'all') {
+    if (id === 'all' || id === 'moderators' || id === 'here') {
```

Comment:
we should maybe constantize these strings since we use them a lot in this file, but it's all good for now :)